packages/expo-cli/src/commands/push-creds.js

```diff
@@ -72,6 +72,128 @@ export default (program: any) => {
 
       await apiClient.deleteAsync(`credentials/push/android/${remotePackageName}`);
 
+      log('All done!');
+    }, true);
+
+  const vapidSubjectDescription =
+    'URL or `mailto:` URL which provides a point of contact in case the push service needs to contact the message sender.';
+
+  program
+    .command('push:web:upload [project-dir]')
+    .description('Uploads VAPID key pair and VAPID subject for web push notifications.')
+    .option('--vapid-pubkey [vapid-public-key]', 'URL-safe base64-encoded VAPID public key.')
+    .option('--vapid-pvtkey [vapid-private-key]', 'URL-safe base64-encoded VAPID private key.')
+    .option('--vapid-subject [vapid-subject]', vapidSubjectDescription)
+    .asyncActionProjectDir(async (projectDir, options) => {
+      if (!options.vapidPubkey || !options.vapidPvtkey || !options.vapidSubject) {
+        throw new Error(
+          'Must specify all three fields (--vapid-pubkey, --vapid-pvtkey, and --vapid-subject) to upload.'
+        );
+      }
+
+      log('Reading project configuration...');
+
+      const {
+        args: { remotePackageName },
+      } = await Exp.getPublishInfoAsync(projectDir);
+
+      log('Logging in...');
+
+      let user = await UserManager.getCurrentUserAsync();
+      let apiClient = ApiV2.clientForUser(user);
+
+      log("Setting VAPID keys on Expo's servers...");
```

Comment:
```suggestion
      log('Setting VAPID keys on Expo's servers...');
```

Reply:
There's `Expo's servers` in there, the `'` forces us to use double quotes instead of single quotes.